projects/cdk/components/dialog-host/dialog-host.style.less

```diff
@@ -4,6 +4,10 @@
     bottom: 0;
     width: 100%;
     height: 0;
+
+    .t-dialog:last-of-type ::ng-deep .t-close {
```

Comment:
Can we do it inside dialog with `:host-context`? This code is in CDK but it acts on a particular kind of dialogs from Core, that's wrong.

Reply:
fixed